tests/notifier/notifiers/inventory_summary_test.py

```diff
@@ -292,6 +292,56 @@ def test_inventory_summary_no_summary_data(self, mock_logger):
         self.assertTrue('no summary data is found'
                         in mock_logger.exception.call_args[0][0])
 
+    def test_get_gsuite_dwd_status_disabled(self):
+        mock_inventory_summary = mock.MagicMock()
+        mock_inventory_summary.return_value = [
+            {'resource_type': 'bucket', 'count': 2},
+            {'resource_type': 'dataset', 'count': 4},
+            {'resource_type': 'folder', 'count': 1},
+            {'resource_type': 'object', 'count': 1},
+            {'resource_type': 'organization', 'count': 1},
+            {'resource_type': 'project', 'count': 2}
+        ]
+
+        mock_service_config = mock.MagicMock()
+
+        inventory_summary_object = inventory_summary\
+            .InventorySummary(mock_service_config, 'abcd')
+
+        expected_get_gsuite_dwd_status_output = False
+
+        test_get_gsuite_dwd_status_output = inventory_summary_object\
+            ._get_gsuite_dwd_status(mock_inventory_summary.return_value)
```

Comment:
The `summary_data` does not have to be mocked.
```
fake_summary_data = [
    {'resource_type': 'bucket', 'count': 2},
    {'resource_type': 'dataset', 'count': 4},
    {'resource_type': 'folder', 'count': 1},
    {'resource_type': 'object', 'count': 1},
    {'resource_type': 'organization', 'count': 1},
    {'resource_type': 'project', 'count': 2}
]

_get_gsuite_dwd_status(fake_summary_data)
```

Reply:
Done!